mlir/test/Dialect/MIOpen/lowering_affix_params_negative.mlir

```diff
@@ -0,0 +1,6 @@
+// XFAIL: *
+// REQUIRES: miopen-driver
+// RUN: miopen-gen -p -x2 -t f16 --perf_config "128,128,2,64,64,2,1,1" | mlir-miopen-driver -miopen-affix-params
+// RUN: miopen-gen -p -x2 -t f16 --perf_config "128,128,2,64,64,16,1,1" | mlir-miopen-driver -miopen-affix-params
```

Comment:
Shouldn't this be in the `mlir-miopen-driver` tests?

Reply:
I'll move the test.
Wondering what's a good guideline to distinguish between `Dialect/MIOpen` and `mlir-miopen-driver` tests?

Reply:
On the other hand, this could go in Dialect/MIOpen

I think the distinction is whether you call `mlopen-gen/mlir-miopen-driver` or `miopen-opt`

Reply:
If you wanted this is a Dialect/MIOpen test, I'd pass in just the conv op with the perf_config attribute set

Reply:
Let's keep this test under `Dialect/MIOpen` as I feel it's more related to the lowering process, not the driver tool itself.

I also revised the test so it doesn't depend on `mlir-miopen-driver`.